provider/pkg/gen/examples/embeds.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016-2021, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package examples
+
+import _ "embed" // Needed to support go:embed directive
+
+//go:embed upstream/deployment.md
+var appsV1DeploymentMD string
+
+// ApiVersionToExample contains Markdown-formatted examples corresponding to a k8s apiVersion.
+var ApiVersionToExample = map[string]string{
+	"kubernetes:apps/v1:Deployment": appsV1DeploymentMD,
```

Comment:
This is awesome!
Thoughts on including a go generate target file which handles the conversion from yaml to the relevant code and generate the markdown etc.? It would help not having to be stuck to whatever version of kube2pulumi we have locally etc.

Reply:
Thanks! Yeah, I think we'll want to automate this along those lines in the future, but I made some manual changes to the output, so I'd rather keep this as a starting point.

Reply:
Could you open a follow-up issue to consider automating this? Especially if we are having to perform manual correction it would be really good to have that generation process automated/reproducible.